00-get-started/01-ingest-kafka-data.md

```diff
@@ -87,17 +97,14 @@ We can query from the materialized view to see the results.
 SELECT * FROM visits_stream_mv;
 ```
 
-The results will look like the following.
+The results will look like the following. Note that the rows do not necessarily follow this order.
 
 ```terminal
- page_id | total_visits | unique_visitors |   last_visit_time
-
----------+--------------+-----------------+---------------------
-
- page1   |            1 |               1 | 2023-06-13 10:07:00
-
- page2   |            3 |               2 | 2023-06-13 10:08:00
-
- page3   |            1 |               1 | 2023-06-13 10:09:00
+ page_id | total_visits | unique_visitors |      last_visit_time
+---------+--------------+-----------------+---------------------------
+ page1   |            2 |               2 | 2023-06-13 10:07:00+00:00
+ page2   |            2 |               2 | 2023-06-13 10:08:00+00:00
+ page3   |            1 |               1 | 2023-06-13 10:09:00+00:00
 ```
 
+For more information, check [Ingest data from Kafka](https://docs.risingwave.com/docs/current/ingest-from-kafka/) from the official documentation.
```

Comment:
Question: How is this tutorial different from this official documentation page?

Reply:
This tutorial is a brief version compared to the official documentation, to have users with zero knowledge get started quickly.